<launch>

    <arg name="use_gripper" default="false"/>

    <node pkg="softdrone_core" type="takeoff_test" name="takeoff_test_node" output="screen">
        <remap from="state" to="/mavros/state"/>
        <remap from="pose" to="/mavros/local_position/pose"/>
        <remap from="target" to="/mavros/setpoint_raw/local"/>
        <remap from="lengths" to="/gripper/lengths"/>

        <param name="dist_threshold" value="0.35"/>
        <param name="total_time" value="5.0"/>
        <param name="hover_duration" value="3.5"/>

        <rosparam param="open_lengths">[125, 125, 184, 184]</rosparam>
    </node>

    <node if="$(arg use_gripper)" pkg="softdrone_core" type="gripper_node" name="gripper" output="screen">
        <remap from="lengths" to="/gripper/lengths"/>
    </node>

</launch>
